docs/src/main/asciidoc/software-transactional-memory.adoc

```diff
@@ -82,25 +177,30 @@ The default container (`org.jboss.stm.Container`) provides support for volatile
 
 <1> You need to tell each Container about the type of objects for which it will be responsible. In this example
     it will be instances that implement the FlightService interface.
-<2> Then you create an instance that implements FlightService. You cannot use it directly at this stage because
-    its operations aren't being monitored by the Container.
-<3> To obtain a managed instance, pass the instance to the `container` to obtain a reference through which you
-    will be able perform transactional operations. This reference can be used safely from multiple threads
-    (provided that each thread uses it in a transaction context - see the next section.
+<2> Then you create an instance that implements FlightService. You should not use it directly at this stage because
```

Comment:
```suggestion
<2> Then you create an instance that implements `FlightService`. You should not use it directly at this stage because
```

Reply:
Must've been there originally because I haven't edited that text yet ...